packages/vscode-common/src/TestHelpers.ts

```diff
@@ -46,6 +46,7 @@ export interface TestHelpers {
   runIntegrationTests(): Promise<void>;
 
   cursorlessTalonStateJsonPath: string;
```

Comment:
Should we return the file system instance instead of keep adding paths here?

Reply:
Let's leave for now and revisit if it gets out of hand